internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,467 @@
+
+type flagsImport struct {
+	file                    string
+	overwrite               bool
+	kafkaApiKey             string
+	schemaRegistryApiKey    string
+	schemaRegistryApiSecret string
+}
+
+type kafkaBinding struct {
+	XConfigs map[string]string `yaml:"x-configs"`
+}
+
+type Message struct {
+	SchemaFormat string      `yaml:"schemaFormat"`
+	ContentType  string      `yaml:"contentType"`
+	Payload      any         `yaml:"payload"`
+	Name         string      `yaml:"name"`
+	Tags         []spec2.Tag `yaml:"tags"`
+}
+
+type Spec struct {
+	Asyncapi   string           `yaml:"asyncapi"`
+	Info       spec2.Info       `yaml:"info"`
+	Servers    spec2.Server     `yaml:"servers"`
+	Channels   map[string]Topic `yaml:"channels"`
+	Components Components       `yaml:"components"`
+}
+
+type Components struct {
+	Messages        map[string]Message `yaml:"messages"`
+	SecuritySchemes any                `yaml:"securitySchemes"`
+}
+
+type Security struct {
+	ConfluentBroker         []any `yaml:"confluentBroker"`
+	ConfluentSchemaRegistry []any `yaml:"confluentSchemaRegistry"`
+}
+
+type MessageRef struct {
+	Ref string `yaml:"$ref"`
+}
+
+type Operation struct {
+	OperationId string                        `yaml:"operationId"`
+	TopicTags   []spec2.Tag                   `yaml:"tags"`
+	OpBindings  spec2.OperationBindingsObject `yaml:"bindings"`
+	Message     MessageRef                    `yaml:"message"`
+}
+
+type Topic struct {
+	Description           string        `yaml:"description"`
+	Publish               Operation     `yaml:"publish"`
+	Subscribe             Operation     `yaml:"subscribe"`
+	XMessageCompatibility string        `yaml:"x-messageCompatibility"`
+	Bindings              TopicBindings `yaml:"bindings"`
+}
+
+type TopicBindings struct {
+	Kafka kafkaBinding `yaml:"kafka"`
+}
+
+func newImportCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import",
+		Short: "Populate cluster using AsyncAPI specification file.",
+		Long:  "Add topics, schemas, and tags from the AsyncAPI YAML specification file to the active Confluent Cloud cluster.",
+		Args:  cobra.NoArgs,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Import an AsyncAPI specification file to populate an existing Kafka cluster and Schema Registry.",
+				Code: "confluent asyncapi import --file spec.yaml",
+			},
+		),
+	}
+
+	c := &command{pcmd.NewAuthenticatedStateFlagCommand(cmd, prerunner)}
+	cmd.RunE = c.asyncapiImport
+	cmd.Flags().String("file", "", "Input file name.")
+	cmd.Flags().Bool("overwrite", false, "Overwrite existing topics with the same name.")
+	cmd.Flags().String("kafka-api-key", "", "Kafka cluster API key.")
+	cmd.Flags().String("schema-registry-api-key", "", "API key for Schema Registry.")
+	cmd.Flags().String("schema-registry-api-secret", "", "API secret for Schema Registry.")
+	cobra.CheckErr(cmd.MarkFlagRequired("file"))
```

Comment:
```suggestion

	cobra.CheckErr(cmd.MarkFlagRequired("file"))

```

Reply:
Can you use `cmd.MarkFlagFilename()` here too? (With possible autocompletions, "yaml" and "yml")